### ISSUE TITLE
OpenGL error, have all drivers installed with dirver booster 5.2

### ISSUE THREAD

**Issue by Jeziu** [FutureXpo/PUBG-RADAR-8K#1]
[LWJGL] GLFW_API_UNAVAILABLE error
        Description : WGL: The driver does not appear to support OpenGL
        Stacktrace  :
                org.lwjgl.glfw.GLFW.nglfwCreateWindow(GLFW.java:1235)
                org.lwjgl.glfw.GLFW.glfwCreateWindow(GLFW.java:1388)
                com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createGlfwWindow(Lwjgl3Application.java:425)
                com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createWindow(Lwjgl3Application.java:372)
                com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:107)
                pubg.radar.ui.GLMap.show(GLMap.kt:117)
                pubg.radar.GameKt.main(Game.kt:58)
Exception in thread "main" com.badlogic.gdx.utils.GdxRuntimeException: Couldn't create window
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createGlfwWindow(Lwjgl3Application.java:428)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.createWindow(Lwjgl3Application.java:372)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.<init>(Lwjgl3Application.java:107)
        at pubg.radar.ui.GLMap.show(GLMap.kt:117)
        at pubg.radar.GameKt.main(Game.kt:58)
AL lib: (EE) alc_cleanup: 1 device not closed
Press any key to continue . . .

**Comment by FutureXpo** [comment]
I have the same problem on VM and second pc, idk how to solve it. If somebody know, how to fix it, I will be very grateful

**Comment by Vbyby** [comment]
try to edit glmap.tk so:

    fun show() {
        val config = Lwjgl3ApplicationConfiguration()
        config.setTitle("")
        **config.useOpenGL3(true, 2, 1)**
        config.setWindowedMode(800, 800)
        config.setResizable(true)
        config.setBackBufferConfig(4, 4, 4, 4, 16, 4, 8)

	**System.setProperty("org.lwjgl.opengl.Display.allowSoftwareOpenGL", "true")**

        Lwjgl3Application(this, config)
    }

and recompile

**Comment by Jeziu** [comment]
mvn clean compile and then mvn package is enough to commit this changes?

**Comment by Jeziu** [comment]
not working

**Comment by aznxfrost** [comment]
This happens if there is no GPU on the VM or GPU virtualization doesn't work.

TLDR.  The radar doesn't render if there is no graphics card.

**Comment by FinalFrag** [comment]
Having the same issue in virtualbox on main PC, and on a laptop with intel HD 3000 graphics :(

**Comment by Vbyby** [comment]
> Having the same issue in virtualbox on main PC, and on a laptop with intel HD 3000 graphics 

> :(

install latest driver
https://downloadcenter.intel.com/product/81500/Intel-HD-Graphics-3000

**Comment by Vbyby** [comment]
> Didnt help. mvn clean compile and then mvn package is enough to commit this changes, isnt it?

yes, only this is enough
try install latest driver on your graphics card

**Comment by FinalFrag** [comment]
Downloaded all the latest drivers (even directx for good measure) on the laptop, but to no avail. Still getting the same error as the topic starter.

I downloaded an opengl checking tool and it says the pc supports opengl 3.1

Not sure what else to try :s
